_pages/programs/seasonal.html

```diff
@@ -0,0 +1,95 @@
+        February 1 and 3, 2021 (see details below).</p>
+    <p><strong>Students:</strong> students age 15-25, minimum Low Intermediate/B1 level of language proficiency
+        (High/Upper Intermediate/B2 level recommended).</p>
+    <p><strong>Learning Objectives &amp; Curriculum:</strong></p>
+    <p>As an English Now! Student, you will be able to</p>
+    <ul>
+        <li>
+            <p><strong>Become a better communicator in English</strong></p>
+            <p>English Now!&rsquo;s small class sizes are ideal for a focus on speaking, listening, and relating well &ndash; in English.</p>
+        </li>
+        <li>
+            <p><strong>Study with engaging curriculum materials</strong></p>
+            <p>Our experienced teachers use
+                theme-based curriculum materials that support student engagement, such as textbooks from <a
+                    href="https://eltngl.com/search/productOverview.do?Ntt=world+english%7C1361740514206490706720190329034505477&amp;N=4294918395+200&amp;Ntk=NGL%7CP_EPI&amp;Ntx=mode+matchallpartial&amp;homePage=false">National
+                    Geographic Learning</a>. All students, low intermediate to advanced, gain confidence in communication
+                skills, improve grammar usage, become better listeners, improve reading skills, and expand their vocabulary.
+                Higher-level students work with more advanced materials and work more on writing and presentation skills.</p>
+        </li>
+        <li>
+            <p><strong>Join our community</strong></p>
+            <p>You&rsquo;ll participate in discussion and activities with our community of students in the U.S. and around the
+                world, both online and in-person, when possible.</p>
+        </li>
+    </ul>
+    <p><strong>Program Format:</strong> Classes are interactive and discussion-based: English Now! instructors connect
+        topics to student experiences and advance student English proficiency with a focus on developing listening and
+        speaking skills for improved communication. Students prepare to participate in each class through relevant
+        reading and listening assignments.</p>
+    <p><strong>Schedule: </strong>Monday through Friday, two hours per day, 10 contact hours per week. Classes are
+        offered at different times of the day, including convenient times for students anywhere in the world. Typical
+        class schedule is 9:30-11:30am, U.S. Eastern time - ask us about options convenient for students in Asia.</p>
+    <p>Our program includes:</p>
+    <ul>
+        <li><strong>Intensive language class (eight hours/week) </strong>that meet four days a week.</li>
+        <li><strong>Enrichment activities (two hours/week),</strong> some co-sponsored with the Washington Center for
+            International Education, English Now!&rsquo;s community service partner. Enrichment activities may include:
```

Comment:
Do we need newlines in these paragraphs?

Reply:
We don't, but VSCode's auto formatter created them. I kind of like them in a way, since you can see all text on the page and make insertions without having to use cmd+f to find words.

Reply:
I generally lean towards having the IDE wrap lines automatically, and would prefer maintaining that standard that we adhere to elsewhere in this site, but it's not really a big deal.